Makefile.Common

```diff
@@ -93,8 +93,8 @@ common/goimports: $(GOIMPORTS)
 common/gofumpt: $(GOFUMPT)
 	$(GOFUMPT) -l -w .
 
-.PHONY: govulncheck
-govulncheck: $(GOVULNCHECK)
+.PHONY: vulncheck
+vulncheck: $(GOVULNCHECK)
 	$(GOVULNCHECK) ./...
```

Comment:
Does this not check all files or only the files within that module?

Reply:
Makefile.Common actually check only files for each module. If you'd like to check all files, you can use make govulncheck in Makefile. That is same with other structure (e.g. make golint/lint)